lib/matplotlib/axis.py

```diff
@@ -759,26 +759,22 @@ def _set_axes_scale(self, value, **kwargs):
 
         Parameters
         ----------
-        value : {"linear", "log", "symlog", "logit", ...} or `.ScaleBase`
-            The axis scale type to apply.
+        value : str or `.ScaleBase`
+            The axis scale type to apply.  Supported string values are "linear", "log",
```

Comment:
Can you somehow integrate a link to https://matplotlib.org/devdocs/api/scale_api.html (maybe directly to the table (?) by adding a rst label before the table). The strings alone give a rough idea, but the link would give the ability to read up un what they mean and which kwargs are supported. You then may or may not leave out the explicit bullet list of classes under kwargs. Alternatively, create a table/bullet list (str name: class) under the value attribute to make the connection here, but still link out for additional context.

Reply:
I like the idea of just linking to the table (which I didn't know was there!)  Having the information in one place means we don't have to worry about keeping two versions in sync.